### ISSUE TITLE
Thorough Accessibility Testing

### ISSUE THREAD

**Issue by CrutchTheClutch** [city-mechanical/city-mechanical-website#43]
When developing, tools such as Lighthouse were used to help determine site accessibility to ensure a pleasant user experience for all users.

However, in depth testing has not been done to ensure all accessibility requirements are met.

In depth testing will commence and should be followed in a drafted Pull Request once issues pertaining to accessibility have been found.
